sources/packages/api/src/auth/guards/institution-location.guard.spec.ts

```diff
@@ -21,6 +22,7 @@ const activateGuard = async (
   const guard = new InstitutionLocationGuard(reflector);
   const user = {} as IInstitutionUserToken;
   user.authorizations = authorizations;
+  user.authorizedParty = AuthorizedParties.institution;
```

Comment:
why so we need this `authorizedParty`, for my info

Reply:
In test cases where we expect the guard to block the execution, it escapes the validation because no clientType is assigned to the user when it is initialized for test cases. So initialized with right client type for unit test.

![image](https://user-images.githubusercontent.com/54600590/155227105-003b59f6-7f1f-4f58-8025-dc9dc63bb9cf.png)